partials/home-featured-products.htm

```diff
@@ -1,9 +1,12 @@
 {% if featuredCollection %}
+  {% set productCounter = 1 %}
   {% for product in featuredCollection.products %}
-    <div class="col-md-4 feature-item">
-        {{ partial('shop-grid-item', {'product':product}) }}
-    </div>
-
+    {% if productCounter < 4 %}
```

Comment:
The productCounter is to limit the display to 3 featured items instead of the default 4 like in the live preview of the theme:
https://lemonstand.com/theme-demo?theme=urbanessence